estate/models/real_estate_property.py

```diff
@@ -0,0 +1,29 @@
+from odoo import fields, models
+
+class TestModel(models.Model):
+    _name = "test_model_real_estate"
```

Comment:
```suggestion
    _name = "test.model.real.estate"
```
generally we define name of models like this instead of using underscore(_)